geopy/geocoders/pickpoint.py

```diff
@@ -0,0 +1,74 @@
+"""
+PickPoint geocoder
+"""
+
+from geopy.compat import urlencode
+from geopy.geocoders import Nominatim
+
+from geopy.geocoders.base import (
+    DEFAULT_FORMAT_STRING,
+    DEFAULT_TIMEOUT
+)
+
+__all__ = ("PickPoint",)
+
+
+class PickPoint(Nominatim):
+    """
+    PickPoint geocoder is a commercial version of Nominatim. Documentation at:
+       https://pickpoint.io/api-reference
+    """
+
+    def __init__(
+            self,
+            api_key,
+            format_string=DEFAULT_FORMAT_STRING,
+            view_box=None,
+            country_bias=None,
+            timeout=DEFAULT_TIMEOUT,
+            proxies=None,
+            user_agent=None
+    ):
+        """
+
+        :param string api_key: PickPoint API key obtained at https://pickpoint.io.
+        :param string format_string: String containing '%s' where the
+            string to geocode should be interpolated before querying the
+            geocoder. For example: '%s, Mountain View, CA'. The default
+            is just '%s'.
+        :param tuple view_box: Coordinates to restrict search within.
+        :param string country_bias: Bias results to this country.
+        :param int timeout: Time, in seconds, to wait for the geocoding service
+            to respond before raising a :class:`geopy.exc.GeocoderTimedOut`
+            exception.
+        :param dict proxies: If specified, routes this geocoder's requests
+            through the specified proxy. E.g., {"https": "192.0.2.0"}. For
+            more information, see documentation on
+            :class:`urllib2.ProxyHandler`.
+        """
+
+        Nominatim.__init__(
```

Comment:
Any reason to not use `super(PickPoint, self).__init__(` here? The base class `Geocoder` is a new-style class (it extends `object`), so it should be fine. Also all other geocoders call constructor this way.

I would also suggest to use keyword arguments instead of positional ones, because this is really a large set of them.